tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "jsx": "react",
+    "lib": [
+      "dom",
+      "es2020"
+    ],
+    "moduleResolution": "bundler",
+    "noEmit": true,
+    "strict": true,
+    "target": "es2020",
+    "typeRoots": [
+      "./node_modules/@types",
+      "./test/types"
+    ]
+  },
+  "include": [
+    "src/**/*"
+  ]
+}
```

Comment:
One small nit:

We do this via an ugly? Dynamic include so it's logical it doesn't know but can we teach it about it?

![image](https://github.com/cockpit-project/cockpit-navigator/assets/67428/0161b280-a8fa-4ee5-8339-4e67efef8fc7)

Reply:
Seems we can do this:

```json
  "compilerOptions": {
    "allowJs": true,
    "checkJs": true,
    "jsx": "react",
    "lib": [
      "dom",
      "es2020"
    ],
    "paths": {
	    "*": ["*", "pkg/lib/*"]
    },
```